package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@tv2media/timeline-state-resolver-packages",
+  "name": "timeline-state-resolver-packages",
```

Comment:
We still need the @tv2media prefix to publish to our NPM repo

Reply:
I don't think we need it here. This file is just for the monorepo/workspace

Reply:
I agree with @ianshade. This package.json is only used to orchestrate the TSR and TSR-types packages.

Reply:
Good point. I think you are right